whelktool/scripts/cleanups/2020/10/lxl-3391-remove-unhandled-375-and-377-from-auth.groovy

```diff
@@ -0,0 +1,38 @@
+PrintWriter scheduledForUpdating = getReportWriter("scheduled-updates")
+PrintWriter failedUpdating = getReportWriter("failed-updates")
+
+String where = "collection = 'auth' and deleted = false and data#>>'{@graph,0,_marcUncompleted}' ~ '\"375\"|\"377\"'"
```

Comment:
This query might also fetch things that you don't necessarily want to process, like (contrived example):
```
      "_marcUncompleted": [
        {
          "700": {
            "ind1": "1",
            "ind2": "0",
            "subfields": [
              {
                "a": "375"
              },
```
This is in itself not a problem, but (see following comment)...